src/smc-webapp/printing.coffee

```diff
@@ -356,6 +358,9 @@ class SagewsPrinter extends Printer
                         data_url = scene.static_image
                         out ?= ''
                         out += "<div class='output sage3d'><img src='#{data_url}'></div>"
+                else if ext == 'webm'
+                    out ?= ''
+                    out += "<video src='#{mesg.file.url}' class='sagews-output-video' controls></video>"
```

Comment:
It's being replaced in `embed_videos` with such a data uri, but only if there isn't a problem getting the file. I wished there is a way to do the same canvas trick as with images, though.